release/nightly_tests/shuffle/shuffle_compute_multi.yaml

```diff
@@ -10,7 +10,7 @@ head_node_type:
 
 worker_node_types:
     - name: worker_node
-      instance_type: i3.8xlarge
+      instance_type: i3.4xlarge
```

Comment:
If we use 8xlarge, it won't use spilling for 100GB shuffle (since 4 * 30 GB object store memory == 120GB)